mypy/checkexpr.py

```diff
@@ -62,6 +62,8 @@
 
 from mypy import experiments
 
+import mypy.type_visitor
```

Comment:
Why do we need this import? (It looks unused, so a comment would help if it is really needed.)

Reply:
Actually whatever it was seems fixed soooo